torch/utils/_foreach_utils.py

```diff
@@ -5,6 +5,59 @@
 from torch import Tensor
 from torch.autograd.grad_mode import no_grad
 
+__all__ = ["SupportForeachDevices"]
+
+class SupportForeachDevices(object):
+    r"""
+    A class that manages the default device type for foreach support,
+    default is List['cuda'].
+    """
+    _default_device_types = ["cuda"]
```

Comment:
For example, this does feel more unsafe, but what do we benefit from the class wrapper?

```suggestion
__all__ = ["foreach_supported_devices"]

foreach_supported_devices = ["cuda"]
```